site/src/components/JsonSchemaPreview.tsx

```diff
@@ -346,6 +382,17 @@ function resolveSchema(schema: JSONSchema7, defs: Record<string, JSONSchema7Defi
     throw new Error('unsupported');
   }
 
+  if (schema.anyOf) {
+    return {
+      ...schema,
+      anyOf: schema.anyOf.map(s => resolveSchema(s as JSONSchema7, defs))
+    };
+  }
+
+  if (schema.additionalProperties) {
+    return resolveSchema(schema.additionalProperties as JSONSchema7, defs);
+  }
```

Comment:
This already gets resolved on line 189. Rendering
```
"builds": {
  "type": "object",
  "additionalProperties": {
    "$ref": "#/definitions/Build"
  }
},
```
should be handled there in order to differentiate between a map and an `additionalProperties` with a `$ref`